tests/ffi/class_in_ns.rs

```diff
@@ -0,0 +1,21 @@
+// To test receivers on a type in a namespace outide
+// the default. cxx::bridge blocks can only have a single
+// receiver type, and there can only be one such block per,
+// which is why this is outside.
+
+#[rustfmt::skip]
+#[cxx::bridge(namespace = tests)]
+pub mod ffi3 {
+
+    extern "C" {
+        include!("tests/ffi/tests.h");
+
+        #[namespace (namespace = I)]
```

Comment:
We should get rid of the stuttering here. I think you mentioned that your IDE was choking on `#[namespace = A::B]` and so that's why you switched to `#[namesapce(namespace = A::B)]`. This is probably the same thing I ran into in #298, and which dtolnay@ filed https://github.com/rust-lang/rust/pull/76734 for (it's not fixed yet, that issue was closed in lieu of another issue). The attribute's value just needs to be quoted (i.e. `#[namespace = "A::B"]`) until that is fixed to allow for unquoted paths as values in name-value attributes. If you take the attribute parsing I wrote in #298 then it should already do the right thing.

Reply:
Addressing in #380 380.